gc/base/GCExtensionsBase.hpp

```diff
@@ -279,8 +279,10 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	bool isArrayletDoubleMapRequested;
 	bool isArrayletDoubleMapAvailable;
 #endif /* OMR_GC_DOUBLE_MAP_ARRAYLETS */
+#if defined(OMR_ENV_DATA64)
```

Comment:
like with OMR_GC_DOUBLE_MAP_ARRAYLETS should we have an explicit flag?

Reply:
A good question, and it is certainly not a bad idea. I will do whatever is decided following the conclusion of [this](https://github.com/eclipse-openj9/openj9/pull/14667#discussion_r1194422140) discussion.